_posts/2020-10-12-sparklyr-flint-0.2.0-released/sparklyr-flint-0.2.0.Rmd

```diff
@@ -0,0 +1,143 @@
+
+So, to summarize, the OLS regression functionality implemented in Flint not only outputs the solution to the problem, but also calculates useful metrics that help data scentists assess the sanity and predictive quality of the resulting model.
+
+To see OLS regression in action with `sparklyr.flint`, one can run the following example:
+
+```
+mtcars_sdf <- copy_to(sc, mtcars, overwrite = TRUE) %>%
+  dplyr::mutate(time = 0L)
+mtcars_ts <- from_sdf(mtcars_sdf, is_sorted = TRUE, time_unit = "SECONDS")
+model <- ols_regression(mtcars_ts, mpg ~ hp + wt) %>% to_sdf()
+
+print(model %>% dplyr::pull(akaikeIC))
```

Comment:
post looks great! NIT: Rather than doing various print methods, is there a clean way to convert the model to a data frame to show all the values at once?

You could consider supporting `broom` in a subsequent release to let the users extract the model parameters through a known package.

Reply:
Actually, `model` is already a data frame, right? SO maybe just print `model`? Unless it has a bunch of unnecessary columns, in which case, maybe `dplyr::select()`?

Reply:
Yes model is something convertible to a Spark dataframe, but it has too many columns.